app/Http/Controllers/TagController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Tag;
+use App\Http\Requests\TagRequest;
+
+class TagController extends Controller
+{
+
+    
+    public function store(TagRequest $request)
+    {
+        $tag = Tag::create([
+            'name' => $request->name,
+        ]);
+
+        return response()->json($tag, 201);
```

Comment:
```suggestion
        return response()->json($tag, 201);
```

Create Laravel resource for that - https://laravel.com/docs/12.x/eloquent-resources